src/components/GameSetup.vue

```diff
@@ -26,10 +28,8 @@ export default defineComponent({
     </div>
   </div>
   <div class="peek">
-    <PeekInside v-if="devMode"/>
+    <PeekInside v-if="devMode" />
```

Comment:
Prettier does not seem to work in my .vue files. I added a bunch of devDependencies according to the documentation of the "Prettier ESlint" extension.

Reply:
Ah, when I choose Volar as "default formatter", it works.